src/deckgl/raster-layer/raster-layer.js

```diff
@@ -90,11 +89,12 @@ export default class RasterLayer extends BitmapLayer {
       }
     }
 
+    const parentShaders = super.getShaders();
     return {
-      ...super.getShaders(),
+      ...parentShaders,
       vs: webgl2 ? vsWebGL2 : vsWebGL1,
       fs: webgl2 ? fsWebGL2 : fsWebGL1,
-      modules: [project32, ...modules],
+      modules: [...parentShaders.modules, ...modules],
```

Comment:
So `project32` is already included in `parentShaders.modules`?

Reply:
Yes https://github.com/visgl/deck.gl/blob/master/modules/layers/src/bitmap-layer/bitmap-layer.js#L52

Reply:
Ok. I can't remember if it's an issue to include the `picking` module, despite not enabling picking through the layer.

Reply:
That's something I want to explore next, if it's possible to support picking. I suppose it wasn't possible yet when you removed it in #32, but later there was this change https://github.com/visgl/deck.gl/pull/5341 so it might be possible already.